src/stories/OSCALControl.stories.js

```diff
@@ -0,0 +1,118 @@
+import React from "react";
+import OSCALControl from "../components/OSCALControl";
+
+export default {
+  title: "Components/Control Display",
```

Comment:
This should probably reflect the component name, so just `Control` rather than `Control Display`.